backend/Models/Database/WhaleSpecies.cs

```diff
@@ -0,0 +1,12 @@
+namespace WhaleSpotting.Models.Database
+{
+    public class WhaleSpecies
```

Comment:
I think this can just be called `Species`

Reply:
I may start a war as it has allready been agreed ?

Reply:
Bumping thread, please do rename!

Reply:
Also remember to rename the file